### PR TITLE
Document socket LB tracing

### PR DESCRIPTION
**Pull request by aditighag**

### CODE REVIEW

File: pkg/cgroups/manager/manager.go [hunk 1]
```diff
@@ -196,9 +196,8 @@ func (m *CgroupManager) enable() {
 		var err error
 		if m.pathProvider, err = getCgroupPathProvider(); err != nil {
 			log.Warn("No valid cgroup base path found: socket " +
-				"load-balancing tracing feature will not work. File a GitHub issue" +
-				"with an example cgroup path for a pod by running command on Kubernetes node: " +
-				"sudo crictl inspectp -o=json $POD_ID | grep cgroupsPath")
+				"load-balancing tracing with Hubble will not work. See" +
+				"the kubeproxy-free guide for more details.")
```
Comment (qmonnet) [inline pkg/cgroups/manager/manager.go]: Nit: Break on sentences?
```suggestion
			log.Warn("No valid cgroup base path found: socket load-balancing tracing with Hubble will not work." +
				"See the kubeproxy-free guide for more details.")
```
Or even write the full message on a single line? Would be long, but not without precedent.

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 2]
```diff
@@ -1399,6 +1399,101 @@ As per `k8s Service <https://kubernetes.io/docs/concepts/services-networking/ser
...
+
+    $ hubble observe --all | grep mediabot
+    Jan 13 13:47:20.932: default/mediabot (ID:5618) <> default/nginx-service:80 (world) pre-xlate-fwd TRACED (TCP)
+    Jan 13 13:47:20.932: default/mediabot (ID:5618) <> default/nginx:80 (ID:35772) post-xlate-fwd TRANSLATED (TCP)
+    Jan 13 13:47:20.932: default/nginx:80 (ID:35772) <> default/mediabot (ID:5618) pre-xlate-rev TRACED (TCP)
+    Jan 13 13:47:20.932: default/nginx-service:80 (world) <> default/mediabot (ID:5618) post-xlate-rev TRANSLATED (TCP)
+    Jan 13 13:47:20.932: default/mediabot:38750 (ID:5618) <> default/nginx (ID:35772) pre-xlate-rev TRACED (TCP)
+
+Socket LB tracing with Hubble requires cilium agent to detect pod cgroup paths.
+If you see a warning message in cilium agent "No valid cgroup base path found: socket load-balancing tracing with Hubble will not work.",
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]:
```suggestion
If you see a warning message in cilium agent ``No valid cgroup base path found: socket load-balancing tracing with Hubble will not work.``,
```

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 3]
```diff
@@ -1399,6 +1399,101 @@ As per `k8s Service <https://kubernetes.io/docs/concepts/services-networking/ser
...
+you can trace packets using ``cilium monitor`` instead.
+
+.. note::
+
+    File a GitHub issue with an example cgroup path for any pod by running command
+    on a Kubernetes node in your cluster: ``sudo crictl inspectp -o=json $POD_ID | grep cgroupsPath``.
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: You mean that the user should file the issue if their cgroup path fails to be detected, right?

```suggestion
    If the cgroup path is not found, please file a GitHub issue with the cgroup path for any of your pods, obtained by running the following command
    on a Kubernetes node in your cluster: ``sudo crictl inspectp -o=json $POD_ID | grep cgroupsPath``.
```

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 4]
```diff
@@ -1399,6 +1399,101 @@ As per `k8s Service <https://kubernetes.io/docs/concepts/services-networking/ser
...
+      selector:
+        app: proxy
+      ports:
+      - port: 80
+
+Deploy a client pod to start traffic.
+
+.. code-block:: shell-session
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-dns/dns-sw-app.yaml
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: This one must be a `parsed-literal` for the substitution on `|SCM_WEB|` to happen.
```suggestion
.. parsed-literal::

    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-dns/dns-sw-app.yaml
```

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks! Looks good, with one formatting issue and a few minor suggestions.

**Review by aspsk — APPROVED** [review]

**Comment by aditighag** [comment]
@qmonnet Thanks! Applied all the suggestions. PTAL.

**Review by qmonnet — APPROVED** [review]
Looks all good, thank you!

**Comment by aditighag** [comment]
Rebased the PR.

**Pull request merged**